docs/sources/shared/datasources/tempo-editor-traceql.md

```diff
@@ -23,19 +23,24 @@ Query results are returned faster because the queries limit what is searched.
 
 To learn more about how to query by TraceQL, refer to the [TraceQL documentation](https://grafana.com/docs/tempo/latest/traceql/).
 
-The TraceQL query editor, located on the **Explore** > **TraceQL** tab in Grafana, lets you search by trace ID and write TraceQL queries using autocomplete.
+The TraceQL query editor **Explore** in Grafana lets you search by trace ID and write TraceQL queries using autocomplete.
 
-![The TraceQL query editor](/media/docs/tempo/traceql/screenshot-traceql-query-editor-v10.png)
+![The TraceQL query editor](/media/docs/grafana/data-sources/tempo/query-editor/tempo-ds-query-traceql-v11.png)
 
-## Enable the query editor
+## Before you begin
 
 This feature is automatically available in Grafana 10 (and newer) and Grafana Cloud.
 
 To use the TraceQL query editor in self-hosted Grafana 9.3.2 and older, you need to [enable the `traceqlEditor` feature toggle](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/feature-toggles/).
 
-If trying to query a self-managed Grafana Tempo or Grafana Enterprise Traces database with a gateway (e.g., nginx) in front of it from your hosted Grafana, that gateway (e.g., nginx) must allow gRPC connections. If it does not, streaming will not work and queries will fail to return results.
+### Streaming and gRPC
 
-If you cannot configure your gateway to allow gRPC, open a support escalation to request streaming query results be disabled in your hosted Grafana.
+If trying to query a self-managed Grafana Tempo or Grafana Enterprise Traces database with a gateway, such as nginx, in front of it from your hosted Grafana, that gateway (for example, nginx) must allow gRPC connections.
+If it doesn't, streaming won't work and queries will fail to return results.
+
+If you can't configure your gateway to allow gRPC, deactivate streaming in your hosted Grafana.
+In Grafana 11.2 and newer, you can deactivate **Streaming** in your Tempo data source settings from **Connections** > **Data sources** in the Grafana menu.
```

Comment:
```suggestion
In Grafana 11.2 and newer, you can disable the **Streaming** option in your Tempo data source settings from **Connections** > **Data sources** in the Grafana menu.
```

Reply:
Aren't we moving away from wording like "enable" and "disable"?

Reply:
My bad. Conflating style guides in my head. 
```suggestion
In Grafana 11.2 and newer, you can deactivate the **Streaming** option in your Tempo data source settings from **Connections** > **Data sources** in the Grafana menu.
```

Reply:
I appreciate all of your comments. Thank you for the thorough review! You've found a bunch of stuff. :)